packages/angular_devkit/build_angular/test/tslint/works_spec_large.ts

```diff
@@ -6,254 +6,282 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import { DefaultTimeout, TestLogger, runTargetSpec } from '@angular-devkit/architect/testing';
-import { normalize, virtualFs } from '@angular-devkit/core';
-import { EMPTY } from 'rxjs';
-import { catchError, tap } from 'rxjs/operators';
-import { Schema as TslintBuilderOptions } from '../../src/tslint/schema';
-import { host, tslintTargetSpec } from '../utils';
+import { WorkspaceNodeModulesArchitectHost } from '@angular-devkit/architect/node';
```

Comment:
This file looks like it has a lot of changes but was actually deleted, and the other deleted file in this PR was renamed to this one. There's actually no code changes.